pkgs/servers/sql/postgresql/libpq.nix

```diff
@@ -0,0 +1,12 @@
+# This is just a thin wrapper around the latest postgresql package. This allows
+# to use this package with properly split dev output - while still making it
+# impossible to accidentally use this as a server package.
+{ postgresql
+, runCommand
+}:
+runCommand "libpq-${postgresql.version}" {
+  outputs = ["out" "dev"];
+} ''
+  ln -s ${postgresql.lib} $out
+  ln -s ${postgresql.dev} $dev
+''
```

Comment:
Should this be a proper package with `pname`, `version`, `meta` so it's discoverable?